### ISSUE TITLE
From Discord: Error in console on snapshot build, item triggers not succeeding

### ISSUE THREAD

**Issue by bitrot-alpha** [RockinChaos/ItemJoin#328]
**Describe the bug**
I'm messing around with the latest version of Minecraft (1.16), which means I have to use dev builds. I'm trying to isolate my Factions plugin to only the relevant parts of the server, just the Factions world, which means I'm moving from Factions Safezone claims to WorldGuard regions.

One of the ways you can trigger an item in this plugin is to set a region trigger. Another way is a world trigger. I tried both, which gave me an error in server console.

**To Reproduce**
- Use a multiworld plugin like MultiVerse
- Set a named worldguard region in each desired world (with interaction flags enabled,etc)
- Use plugin GUI to create an item with triggers for named Worldguard regions, and/or the worlds containing said regions.
- Teleport somehow to place where you expect triggers to occur
- Item does not appear!

**Expected behavior**
Main function of the plugin is to give players items when specified triggers occur. Hit-or-miss with dev builds right now (understandably so).

**Pastebin**
[Console error](https://pastebin.com/EgT1D1gq)
[items.yml](https://pastebin.com/zR6hjTBL)
[config.yml](https://pastebin.com/3dBYHauY)

**Versions:**
 - Resource Version and Build Number: `5.0.7-b511`
 - Server Version `Paper-1.16.1 #136`

**Additional context**
```
Plugins (43): AntiDisconnectSpam, BlockLocker, Bookshelves, Cannons, ChatEx*, ChestShop, ChestSort, ClearLag,
CoreProtect, dynmap*, Essentials, ExecutableItems, Factions, floodgate-bukkit, Geyser-Spigot, GSit,
IridiumSkyblock, ItemJoin, LagMonitor, LoginSecurity*, LuckPerms, Multiverse-Core, Multiverse-NetherPortals,
Multiverse-Portals, PerWorldInventory, PlaceholderAPI, Plan, PlotSquared, PlugMan, ProtocolLib, ServerListPlus*,
Ships, SkinsRestorer, UberEnchant, Vault, ViaBackwards, ViaRewind, ViaVersion, VoidGenerator, WarpSystem,
WorldBorder, WorldEdit, WorldGuard
```
Posted error on Discord, but submitting bug report for sake of being thorough.

**Comment by RockinChaos** [comment]
Should now be fixed in the latest snapshot; http://ci.craftationgaming.com/job/ItemJoin/531

Let me know!

**Comment by RockinChaos** [comment]
This bug has been officially fixed in the release of ItemJoin v5.0.7.
You can grab it here; Download: https://www.spigotmc.org/resources/itemjoin.12661/download?version=354306

Please read the changelog as the documentation has not been updated yet;
https://github.com/RockinChaos/ItemJoin/wiki/Recent-Changes
